sdks/java/core/src/main/java/org/apache/beam/sdk/io/PubsubIO.java

```diff
@@ -253,6 +257,28 @@ public String asPath() {
   }
 
   /**
+   * Simple {@link SerializableFunction} for {@link PubsubTopic}.
+   */
+  public static class TopicTranslator
```

Comment:
Esp if visibility is reduced, could improve javadoc to be something like "Used to build a {@link ValueProvider} for {@link PubsubTopic}"